app/src/main/java/com/example/wellfed/recipe/RecipeAdapter.java

```diff
@@ -31,12 +31,20 @@ public class RecipeAdapter extends RecyclerView.Adapter<RecipeAdapter.ViewHolder
     }
 
 
-    public RecipeAdapter(FragmentActivity parent, List<Recipe> recipes, RecipeBookFragment recipeBookFragment) {
+    public RecipeAdapter(FragmentActivity parent, List<Recipe> recipes,
+                         FragmentActivity recipeBookFragment) {
         this.parent = parent;
         this.recipes = recipes;
         this.recipeLauncher = (RecipeLauncher) recipeBookFragment;
     }
 
+    // TODO: make Adapter callable by any fragment
```

Comment:
Want to reuse the adapter in MealPlan activity

Reply:
It's going to use same adapter, but different list of recipes and different viewholder

Reply:
> Want to reuse the adapter for the list of recipes in in MealPlanActivity

can you please elaborate on what user stories exactly?

Reply:
as a user i will want to add a recipe to my meal plan, i cannot think of the flow it will work.
maybe a discussion issue on how this will operate? user wants to add a recipe -> a list of recipes are displayed -> user chooses a recipe? i think i get it, correct me if you have some other plan